db/seeds.rb

```diff
@@ -1,16 +1,46 @@
-# Создадим записи в базе данных для проверки,
-# восклицательный знак после create стоит для вызова исключения.
+# encoding: utf-8
+require 'open-uri'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.